spec/services/anonymise_user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe AnonymiseUser do
+  let(:user) { create(:administrator) }
+
+  describe "#call" do
+    it "returns a successful result" do
+      result = described_class.new(user: user).call
+
+      expect(result.success?).to be(true)
+      expect(result.failure?).to be(false)
```

Comment:
I guess this is local customs. I'd usually write

```suggestion
      expect(result).to be_success
      expect(result).not_to be_failure
```